.github/renovate-self-host-config.json5

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Self-hosted options (https://docs.renovatebot.com/self-hosted-configuration/)
+{
+  // Temporary during validation
+  "dryRun": true,
```

Comment:
where does this log to? I didn't see any configurations that specify that?

Reply:
It should log to the GitHub workflow's log output